src/dodal/devices/pressure_jump_cell.py

```diff
@@ -163,6 +144,44 @@ async def set(self, value: AllValvesControlState):
         )
 
 
+class ValveControl(StandardReadable):
+    def __init__(self, prefix: str, name: str = "") -> None:
+        with self.add_children_as_readables():
+            self.close = epics_signal_rw(ValveControlRequest, prefix + ":CON")
+            self.open = epics_signal_rw(ValveOpenSeqRequest, prefix + ":OPENSEQ")
+
+        super().__init__(name)
+
+    def set(self, value: ValveControlRequest | ValveOpenSeqRequest) -> AsyncStatus:
+        setStatus = None
+
+        if isinstance(value, ValveControlRequest):
+            setStatus = self.close.set(value)
+        elif isinstance(value, ValveOpenSeqRequest):
+            setStatus = self.open.set(value)
```

Comment:
To get the momentary signal I mentioned in https://github.com/DiamondLightSource/dodal/pull/812#issue-2564153769 could do something like `asyncio.sleep(TRIGGER_TIME)` and then setting it back to INACTIVE but this wouldn't stop the the value being directly set to `OPEN_SEQ` holding it at a value. 

Is there a dodal mechanism for momentary signals to replicate what the synoptic screen button is doing?

Reply:
Implemented as delay in set_valve(), fixed in c46fb8e3992044181b8f7dd038e140ca59791d5a

Reply:
good question for the momentary signals. Let's keep this PR as-is.

[I don't see any prior discussion](https://github.com/DiamondLightSource/dodal/issues?q=is%3Aissue+temporary) on this kind of mechanism - if this is needed more broadly we might want to standardize this. 

Please make a new issue as `question` to discuss this, but it's low priority atm

Reply:
Created issue #942 for momentary signals.